src/tools/web/elektrad/README.md

```diff
@@ -1,30 +1,25 @@
 # elektra-web/elektrad
 
-A server that provides an [HTTP API](http://docs.elektrad.apiary.io/) to access
-Elektra remotely, built using [express](https://expressjs.com/).
+A server that provides an [HTTP API](http://docs.elektrad.apiary.io) to access
+Elektra remotely, built using [go](https://golang.org).
 
 ## Installation
 
-- first, install [libelektra](https://libelektra.org/)
-- now run `npm install`
+- make sure you have go (>1.11) installed.
+- install [libelektra](https://libelektra.org/), make sure the package-config env variable is set correctly.
+- now run `go build` in the elektrad folder with go module enabled (GO111MODULE=on).
```

Comment:
Why isn't this done during the build process?

Reply:
This is already done during the build process. These instructions are for manually compiling the server without cmake for e.g. debugging purposes.

Reply:
This should be clearly said.